src/client/dfs/duns.c

```diff
@@ -970,3 +1001,16 @@ duns_destroy_path(daos_handle_t poh, const char *path)
 
 	return 0;
 }
+
+int
+duns_destroy_attr(struct duns_attr_t *attrp)
+{
+	if (attrp == NULL)
+		return EINVAL;
```

Comment:
nit: Does this function really need to return a status? If the pointer is null, maybe just return early. Then the caller doesn't need to check a rc to determine that it shouldn't have called the function in the first place, or else ignore the rc which is bad form.

Reply:
for now right, im just wondering if we add new fields in the future, like pool or container handles that need to be closed, and we would need to propagate the error in this case.

Reply:
maybe for those we require explicit close / disconnect on the handles if we ever add them. i'll make the change.